index.js

```diff
@@ -1497,6 +1497,26 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# both :: (a -> Boolean) -> (a -> Boolean) -> a -> Boolean
+  //.
+  //. Takes two unary functions that return a boolean and applies them
+  //. both to the supplied value, returning true iff both functions
+  //. evaluate to true.
```

Comment:
- Let's write _unary predicates_ rather than _unary functions that return a boolean_.
  - It's not accurate to state _applies them both_ (due to short-circuiting).

How about the following?

_Takes two unary predicates and a value of any type, and returns `true` if the value satisfies both predicates; `false` otherwise._